diff --git a/mmaudio/ext/bigvgan_v2/bigvgan.py b/mmaudio/ext/bigvgan_v2/bigvgan.py
--- a/mmaudio/ext/bigvgan_v2/bigvgan.py
+++ b/mmaudio/ext/bigvgan_v2/bigvgan.py
@@ -364,12 +364,12 @@ class BigVGAN(PyTorchModelHubMixin, nn.Module):
         revision: str,
         cache_dir: str,
         force_download: bool,
-        proxies: Optional[Dict],
-        resume_download: bool,
         local_files_only: bool,
         token: Union[str, bool, None],
+        proxies: Optional[Dict] = None,
+        resume_download: bool = False,
         map_location: str = "cpu",
         strict: bool = False,
         use_cuda_kernel: bool = False,
